Makefile

```diff
@@ -9,46 +9,68 @@ CRONDIR       ?= $(SYSCONFDIR)/cron.d
 LOGDIR        ?= /var/log
 
 all:
-	@echo "Nothing to compile, run make install."
+	@echo "Nothing to compile, please choose a specific target:"
+	@echo "    install (includes install-base, install-man, and install-doc)"
+	@echo "    install-base"
+	@echo "    install-man"
+	@echo "    install-doc"
+	@echo "    uninstall (includes uninstall-base, uninstall-man, and uninstall-doc)"
+	@echo "    uninstall-base"
+	@echo "    uninstall-man"
+	@echo "    uninstall-doc"
 
-install:
+src/recap.cron:
+	@sed -e 's|@BINDIR@|$(BINDIR)|' src/recap.cron.in > src/recap.cron
```

Comment:
As a note, if this is merged it will break the `recap-installer` in #90 the attempt is to remove the installer as the `Makefile` should be able to replace it. Mentioning this as we need coordination to prevent breaking the installer.